runtime/compiler/env/j9method.cpp

```diff
@@ -8008,7 +8008,7 @@ TR_J9ByteCodeIlGenerator::runFEMacro(TR::SymbolReference *symRef)
          // Construct the signature string for the array class
          //
          UDATA arity = arrayJ9Class->arity;
-         J9Class *leafClass = arrayJ9Class->leafComponentType;
+         J9Class *leafClass = (J9Class*)fej9->getLeafComponentClassFromArrayClass((TR_OpaqueClassBlock*)arrayJ9Class);
```

Comment:
I'm honestly not sure if this ends up getting executed during AOT (I haven't seen it executed myself). I noticed this while fixing the other issue and figured I'd change this to use the front-end API pre-emptively to prevent any unwanted surprises if this ever does end up getting executed during AOT.